src/components/Message/styles.scss

```diff
@@ -9,7 +9,7 @@
 
 .tw-message {
   &.tw-message--page {
-    width: 584px;
+    width: 100%;
```

Comment:
this might break the web-app, tried 100% in chrome inspector and saw this

<img width="956" alt="Terraware App 2022-10-31 13-55-58" src="https://user-images.githubusercontent.com/1865174/199109119-3f05cad2-8c14-47f2-bcf1-a02c5d4a6cc9.png">

Reply:
I was anticipating Hannah's changes to Page Message (https://www.figma.com/file/aLIG1C6uIMyBWiTopSluxe/Visual-Refresh?node-id=880%3A339230) and made it ahead of time. Hopefully won't be an issue...

Reply:
ah nice, i see it now.